pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
```

Comment:
Hopefully someone with more experience than me with Ingress resources can weigh in as well